phpunit/blocks/render-post-template-test.php

```diff
@@ -122,7 +122,7 @@ public function test_rendering_post_template_with_main_query_loop_already_starte
 		global $wp_query, $wp_the_query;
 
 		// Query block with post template block.
-		$content  = '<!-- wp:query {"query":{"inherit":true}} -->';
```

Comment:
Why does this test need to change? Was this not testing with the correct `inherit` setting before?

Reply:
> Was this not testing with the correct inherit setting before?

No 🙈 And it was passing because I was using `query_posts()`, which I think was replacing the global query, so anything set via the Query Loop block was ignored and wasn't truly being tested.

This test should have been updated when the [changes were added](https://github.com/WordPress/gutenberg/pull/65067/files#diff-17253f75062593a5e244507f79cfaa0503bd2d232f64cc66298fbc9346536278R111-R113) for automatically setting `inherit` to `false` in the original PR. It was a complete oversight by me, although I'm glad I updated the test earlier on in the process and so I knew how to fix it. 😅

Reply:
I see! Good thing you caught it early.